python-checks/src/test/java/org/sonar/python/checks/CheckUtilsTest.java

```diff
@@ -170,6 +223,17 @@ private static FileInput parseFile(String path) throws IOException {
     }
   }
 
+  private static FileInput parseFileWithSymbols(String path) throws IOException {
+    var file = new File(path);
+    var baseDirFile = new File(file.getParent());
+
+    var projectLevelSymbolTable = TestPythonVisitorRunner.globalSymbols(List.of(file), baseDirFile);
+    var context = TestPythonVisitorRunner.createContext(file, null, pythonPackageName(file, baseDirFile.getAbsolutePath()), projectLevelSymbolTable,
+      CacheContextImpl.dummyCache());
+
+    return context.rootTree();
```

Comment:
For those tests, I don't think a project symbol table is needed. I would rewrite it as:
```suggestion
return TestPythonVisitorRunner.createContext(new File(path)).rootTree();
```